satpy/etc/readers/omps_edr.yaml

```diff
@@ -166,4 +170,74 @@ datasets:
     units: degrees
     standard_name: latitude
 
+  Longitude:
+    name: Longitude
+    resolution: 50000
+    file_type: omps_sampo
+    file_key: GEOLOCATION_DATA/Longitude
+    units: degrees
+    standard_name: longitude
+  Latitude:
+    name: Latitude
+    resolution: 50000
+    file_type: omps_sampo
+    file_key: GEOLOCATION_DATA/Latitude
+    units: degrees
+    standard_name: latitude
+
+  ColumnAmountO3:
+    name: ColumnAmountO3
+    resolution: 50000
+    coordinates: [Longitude, Latitude]
+    file_type: omps_sampo
+    file_key: SCIENCE_DATA/ColumnAmountO3
+
+  ColumnAmountSO2_PBL:
+    name: ColumnAmountSO2_PBL
+    resolution: 50000
+    coordinates: [Longitude, Latitude]
+    file_type: omps_sampo
+    file_key: SCIENCE_DATA/ColumnAmountSO2_PBL
+
+  ColumnAmountSO2_STL:
+    name: ColumnAmountSO2_STL
+    resolution: 50000
+    coordinates: [Longitude, Latitude]
+    file_type: omps_sampo
+    file_key: SCIENCE_DATA/ColumnAmountSO2_TRL
+
+  ColumnAmountSO2_TRL:
+    name: ColumnAmountSO2_TRL
+    resolution: 50000
+    coordinates: [Longitude, Latitude]
+    file_type: omps_sampo
+    file_key: SCIENCE_DATA/ColumnAmountSO2_TRL
+
+  ColumnAmountSO2_TRM:
+    name: ColumnAmountSO2_TRM
+    resolution: 50000
+    coordinates: [Longitude, Latitude]
+    file_type: omps_sampo
+    file_key: SCIENCE_DATA/ColumnAmountSO2_TRM
+
+  ColumnAmountSO2_TRU:
+    name: ColumnAmountSO2_TRU
+    resolution: 50000
+    coordinates: [Longitude, Latitude]
+    file_type: omps_sampo
+    file_key: SCIENCE_DATA/ColumnAmountSO2_TRU
+
+  UVAerosolIndex:
+    name: UVAerosolIndex
+    resolution: 50000
+    coordinates: [Longitude, Latitude]
+    file_type: omps_sampo
+    file_key: SCIENCE_DATA/UVAerosolIndex
+
+  CloudFraction:
+    name: CloudFraction
+    resolution: 50000
+    coordinates: [Longitude, Latitude]
+    file_type: omps_sampo
+    file_key: SCIENCE_DATA/CloudFraction
```

Comment:
Also about consistency, should the dataset have same case and wording as the ones from the other files ?